pkgs/applications/terminal-emulators/darktile/default.nix

```diff
@@ -60,6 +60,6 @@ stdenv.mkDerivation rec {
     downloadPage = "https://github.com/liamg/darktile/releases";
     changelog = "https://github.com/liamg/darktile/releases/tag/v${version}";
     license = licenses.mit;
-    maintainers = with maintainers; [ flexagoon ];
+    maintainers = with maintainers; [ sogreatandpowerfull ];
```

Comment:
@flexagoon can you change it to `sogreatandpowerful`

Reply:
Sure, I'll change it

Reply:
Thank you!